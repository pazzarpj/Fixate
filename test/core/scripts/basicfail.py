from fixate.core.common import TestClass
from fixate.core.ui import user_ok, user_info
from fixate.core.checks import *

__version__ = "1"


class SimpleTest(TestClass):
    """Simple passing test"""
    retry_type = TestClass.RT_FAIL

    def test(self):
        chk_true(False, "It is True!")


TEST_SEQUENCE = [SimpleTest()]