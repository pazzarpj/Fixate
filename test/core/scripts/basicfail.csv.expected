0,Sequence,started=20181010-070721,fixate-version=0.4.0dev,test-script-name=basicfail,report-format=3,index_string=None
0.00,Test 1,start,Simple passing test,
0.00,Test 1,check1,true,It is True!,FAIL,False
0.00,Test 1,end,FAIL,checks-passed=0,checks-failed=1
0.08,Test 1,waiting
0.08,Sequence,ended=20181010-070723,Finished,tests-passed=0,tests-failed=1,tests-error=0,tests-skipped=0,sequence=FAILED